src/CSVMsg.chpl

```diff
@@ -300,7 +296,6 @@ module CSVMsg {
         var row_ct: int = lines.size - data_start_offset;
 
         reader.close();
-        csvFile.close();
```

Comment:
here we keep the `reader.close()`, I was just wanted to point out the fact that they're different in case it matters